.drone.yml

```diff
@@ -24,3 +24,22 @@ pipeline:
         target: github_release_api_key
     when:
       event: tag
+
+ publish_latest:
```

Comment:
oh oops, guess that's what happens when I hurry with vim :) I'll have that fix in a sec